### ISSUE TITLE
"make test" target is broken

### ISSUE THREAD

**Issue by fjammes** [operator-framework/operator-sdk#4856]
#### What did you do?

Update my Makefile according to "https://sdk.operatorframework.io/docs/building-operators/golang/references/envtest-setup/"
and run "make test"

#### What did you expect to see?

Unit tests should pass.

#### What did you see instead? Under which circumstances?

I get an error message:

```
$ make test
home/fjammes/go/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
goimports -w api
go fmt ./...
go vet ./...
/home/fjammes/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
goimports -w api
mkdir -p /home/fjammes/src/qserv-operator/testbin
test -f /home/fjammes/src/qserv-operator/testbin/setup-envtest.sh || curl -sSLo /home/fjammes/src/qserv-operator/testbin/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
source /home/fjammes/src/qserv-operator/testbin/setup-envtest.sh; fetch_envtest_tools /home/fjammes/src/qserv-operator/testbin; setup_envtest_env /home/fjammes/src/qserv-operator/testbin; go test ./... -coverprofile cover.out
/home/fjammes/src/qserv-operator/testbin/setup-envtest.sh: line 1: 404:: command not found
/bin/bash: fetch_envtest_tools: command not found
/bin/bash: setup_envtest_env: command not found
```
A script is missing here:
```
$ curl -sSLo /home/fjammes/src/qserv-operator/testbin/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
$ cat testbin/setup-envtest.sh 
404: Not Found%                 
```

#### Environment

**Operator type:**

/language go

**Kubernetes cluster type:**

kind

`$ operator-sdk version`
operator-sdk version: "v1.5.0", commit: "98f30d59ade2d911a7a8c76f0169a7de0dec37a0", kubernetes version: "1.19.4", go version: "go1.15.5", GOOS: "linux", GOARCH: "amd64"

`$ go version` (if language is Go)
go version go1.16.3 linux/amd64

`$ kubectl version`
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-25T14:58:59Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-21T01:11:42Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}

#### Possible Solution
Update documentation at https://sdk.operatorframework.io/docs/building-operators/golang/references/envtest-setup/

Possible temporary hack: in documenation, use file https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh instead of https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh

**Comment by burmanm** [comment]
This is the PR that broke it: https://github.com/kubernetes-sigs/controller-runtime/pull/1488

**Comment by estroz** [comment]
Fixed in #4835. See the [migration guide](https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.7.0/#gov3-set-the-makefiles-shell-variable-to-the-systems-bash-binary) for details.
